localization/english/wc_game_concepts_l_english.yml

```diff
@@ -35,4 +35,18 @@
  
  game_concept_grim_religion:0 "Grim Religion"
  game_concept_grim_religions:0 "Grim Religions"
- game_concept_grim_religion_desc:0 "A [religion|E] so dark and evil that it opposes the rest of the world. They don't seek outside allies and won't stop until they reach their ultimate goal.\n\nThese religions are:\n$EFFECT_LIST_BULLET$$deathbringer$\n$EFFECT_LIST_BULLET$$doombringer$\n$EFFECT_LIST_BULLET$$shathgral$"
\ No newline at end of file
+ game_concept_grim_religion_desc:0 "A [religion|E] so dark and evil that it opposes the rest of the world. They don't seek outside allies and won't stop until they reach their ultimate goal.\n\nThese religions are:\n$EFFECT_LIST_BULLET$$deathbringer$\n$EFFECT_LIST_BULLET$$doombringer$\n$EFFECT_LIST_BULLET$$shathgral$"
+ 
+ game_concept_dark_authority:0 "Dark Authority"
+ game_concept_dark_authority_desc:0 "Replaces [crown_authority|E] for [dark_governments|E].\n\nThe $game_concept_dark_authority$ laws change automatically when they accumulate some amount of [dark_frenzy|E] and thus can't be changed manually.\n\nHigh levels of $game_concept_dark_authority$ gives [tax|E], [levy|E], [development|E] bonuses but [vassals|E] dislike it."
```

Comment:
> Replaces [crown_authority|E] with [dark_governments|E]

Not with. The Dark Authority laws replace the Crown Authority for Dark Governments.

Reply:
Afaik "replace" goes with "with sth." or "by sth.", but not with "for sth.".

Reply:
> Afaik "replace" goes with "with sth." or "by sth.", but not with "for sth.".

I have to google English grammar terminology but here we go.

For isn't related to the replaces. The replaces is a verb, the `crown_authority` is an object, the subject is outside the sentence.

The for marks a beginning of coordinating conjunction: `for [dark_governments|E]`.

Other examples:

> It conducts to replace contractors for long-term charter of aircraft.

> It would replace and standards for desktops interfacing with the new system.

Reply:
Ok, thanks for the clarification, shouldn't have started this last evening/night. Forget my suggestion to "replace"...
```suggestion
 game_concept_dark_authority_desc:0 "Replaces [crown_authority|E] for [dark_governments|E].\n\nThe $game_concept_dark_authority$ laws change automatically when enough [dark_frenzy|E] is accumulated and thus can't be changed manually.\n\nHigh levels of $game_concept_dark_authority$ grant [tax|E], [levy|E] and [development|E] bonuses, but [vassals|E] dislike it."
```